functions/Set-DbaBackupPath.ps1

```diff
@@ -0,0 +1,149 @@
+            Copyright (C) 2016 Chrissy LeMaire
+            License: GNU GPL v3 https://opensource.org/licenses/GPL-3.0
+
+        .LINK
+            https://dbatools.io/Set-DbaBackupPath
+
+        .EXAMPLE
+            Set-DbaBackupPath -SqlInstance 'sqlserver1' -Path 'Q:\Backups'
+
+            Set the backup path to Q:\Backups for one server named "sqlserver1"
+
+        .EXAMPLE
+            Get-DbaRegisteredServer -SqlInstance sqlserver | Set-DbaBackupPath 'Q:\Backups'
+
+            Find all servers in SQL Server Central Management server, then pipe those to Set-DbaBackupPath
+            and set the backup directory to Q:\Backups.
+    #>
+    [CmdletBinding(SupportsShouldProcess = $true, ConfirmImpact = 'Medium')]
+    param (
+        [Parameter(Mandatory = $true, ValueFromPipeline = $true)]
+        [Alias("ServerInstance", "SqlServer", "SqlServers", "ComputerName")]
+        [DbaInstanceParameter[]]$SqlInstance,
+        [Alias("Credential")]
+        [PSCredential]$SqlCredential,
+        [Parameter(Mandatory = $true)]
+        [string]$Path,
+        [Alias('Silent')]
+        [switch]$EnableException
+    )
+
+    process {
+        if (Test-FunctionInterrupt) { return }
+
+        foreach ($instance in $SqlInstance) {
+            try {
+                Write-Message -Level Verbose -Message "Connecting to $instance"
+                $server = Connect-SqlInstance -SqlInstance $instance -SqlCredential $SqlCredential
+            }
+            catch {
+                Stop-Function -Message "Failure" -Category ConnectionError -ErrorRecord $_ -Target $instance -Continue
+            }
+
+            if (!(Test-SqlSa -SqlInstance $server -SqlCredential $SqlCredential)) {
+                Stop-Function -Message "Not a sysadmin on $server. Skipping." -Category PermissionDenied -ErrorRecord $_ -Target $server -Continue
+            }
+
+            $createDirectoryBlock = {
+                param ($ServiceAccount, $Path)
+
+                $pathExists = Test-Path -Path $Path
+
+                if (-not ($pathExists)) {
+                    $null = New-Item -Path $Path -ItemType 'Directory'
+
+                    $acl = Get-Acl -Path $Path
+                    $rule = New-Object System.Security.AccessControl.FileSystemAccessRule($ServiceAccount, 'FullControl', 'Allow')
+                    $acl.SetAccessRule($rule)
+                    Set-Acl -Path $Path -AclObject $acl
+                }
+            }
+            $createDirectoryParams = @{
+                ScriptBlock = $createDirectoryBlock
+                ArgumentList = $server.ServiceAccount, $Path
+            }
+
+            $isUnc = ([System.Uri]$Path).IsUnc
+            if (-not $isUnc) {
+                Write-Message -Level 'Verbose' -Message 'Local path found, setting from remote session'
+
+                $createDirectoryParams['ComputerName'] = $server.NetName
+            }
+
+            try {
```

Comment:
This should have a should process around it to let them know in a whatif that a directory would be created.
That would look something like this:
```powershell
if ($Pscmdlet.ShouldProcess("<target>", "Creating ___ on ___)) {...
```